ccmlib/common.py

```diff
@@ -328,6 +328,13 @@ def is_win():
     return sys.platform in ("cygwin", "win32")
 
 
+def is_modern_windows_install(version):
```

Comment:
Could you add a docstring to this function, explaining what it is? Can be as brief as:
```
"""
The 2.1 release line was when Cassandra received beta windows support. Many features are gated based on that added compatibility.
"""
```